apps/ensindexer/src/config/types.ts

```diff
@@ -28,101 +26,7 @@ export interface RpcConfig {
 /**
  * The complete runtime configuration for an ENSIndexer instance.
  */
-export interface ENSIndexerConfig {
```

Comment:
All configuration properties removed from this interface have been moved in to `ENSIndexerPublicConfig` interface in ENSNode SDK package.